deploy/cr.yaml

```diff
@@ -57,6 +57,20 @@ spec:
 #          bucket: "my-bucket"
 #          endpoint: "s3.ca-central-1.amazonaws.com"
 #          region: "ca-central-1"
+#    volumes:
```

Comment:
Can we set custom annotations and labels for this job?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/758/commits/c10c4b74bab04a316b46c61863de161f425c0c15